README.md

```diff
@@ -35,7 +35,7 @@ vehicles_infos = {
   "total_work_time": total work duration available for this vehicle, 0 if all vehicles are the same,
   "total_work_days": total number of days this vehicle can work
 }
-c.vehicles_infos = vehicles_infos
+c.vehicle_infos = vehicle_infos
```

Comment:
The word info is already plural in the sense that you cannot say informationS in English.

Plus a single object is called vehicle_info (i.e., everything related one vehicle) and the array which has several "vehicle_info"s is called vehicle_infos because it contains multiple objects of vehicle_info. Double plural sounds incorrect in English.